python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -1123,6 +1130,40 @@ def configure_logging(self, obj: ConfigureLoggingReq):
         logging.info(f"Config logging: {obj=}")
         self.log_request_metadata = self.get_log_request_metadata()
 
+    def _create_request_metrics_exporters(self) -> List[RequestMetricsExporter]:
```

Comment:
How about moving theses code to the newly added request_metrics_exporter.py file, add a "manager" to manage the exporters? It would minimize the changes in tokenizer_manager.py.